### PR TITLE
QA908 extend longevity coverage for azure

### PR DESCRIPTION
**Pull request by Mark-Gurevich**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: jenkins-pipelines/longevity-1tb-7days-azure.jenkinsfile
Comment (fgelcer) [inline jenkins-pipelines/longevity-1tb-7days-azure.jenkinsfile:1]: need to change file name to reflect this new job being a 5 days long, and not 7 (i know you copied from another job that had that in name, but no one bothered yet to change it to 5 days, as it requires changing multiple Jenkins jobs)

**Review by fgelcer — CHANGES_REQUESTED** [review]

**Comment by fgelcer** [comment]
@Mark-Gurevich , few things:

1. add here the link to the large partition you created
2. check why the checks failed
3. let's start running part of the nemesis jobs, but we won't be able to run all of them in parallel, so let's get some 2, and start running them ASAP

**Comment by fgelcer** [comment]
@Mark-Gurevich , why is it still a draft?
we need to merge and backport this ASAP

**Comment by Mark-Gurevich** [comment]
> @Mark-Gurevich , why is it still a draft? we need to merge and backport this ASAP

it was draft because you told me to hold on with running these tests until we finish with individual nemesis.
Basically, we can merge it and run the tests from the official builds.

File: jenkins-pipelines/longevity-1tb-5days.jenkinsfile
Comment (fgelcer) [inline jenkins-pipelines/longevity-1tb-5days.jenkinsfile:1]: once this one will be merged, you need to fix the calls for it on master and enterprise jobs.
once this one will be backported to 5.2 and 2023.1 you need to fix the respective jobs to use the renamed jenkinsfile

**Review by fgelcer — DISMISSED** [review]
LGTM

**Comment by fgelcer** [comment]
ping @roydahan

File: test-cases/longevity/longevity-large-partition-4days.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-large-partition-4days.yaml:1]: replace this longevity with longevity large-partitions-8h

**Review by fgelcer — APPROVED** [review]
LGTM

**Comment by fgelcer** [comment]
ping @roydahan

File: test-cases/longevity/longevity-large-partition-4days.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-large-partition-4days.yaml:1]: This one should had been removed, but not critical for now.

**Review by roydahan — APPROVED** [review]

**Comment by roydahan** [comment]
@fgelcer run the job that creates these jobs for master, 5.2 and 2023.1.

Let's start running them 1 for 5.2 and the rest on 2023.1

**Comment by fgelcer** [comment]
@roydahan , we can run it, but we need this PR to be backported to the relevant branches.
@scylladb/qa-maintainers

**Comment by fgelcer** [comment]
> @fgelcer run the job that creates these jobs for master, 5.2 and 2023.1.
> 
> Let's start running them 1 for 5.2 and the rest on 2023.1

creating for [5.2](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/create-sct-test-jobs-for-branch/176/) and for [2023.1](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/create-sct-test-jobs-for-branch/177/)

**Comment by soyacz** [comment]
backported

**Pull request merged**
